.envrc

```diff
@@ -0,0 +1,2 @@
+use flake . --accept-flake-config
+dotenv_if_exists .env.local
```

Comment:
Please remove these lines; we don't want to *force* folks to use flakes by default just becaues they may have direnv installed.

Reply:
Fix.
I thought it was fine because `dotenv_if_exists .env.local` is provided for normal users, but as you say, there are people who use direnv but don't use nix.
I used to be one and forgot about it.